### DISCUSSION TITLE
分阶段分析构建耗时

### DISCUSSION THREAD

**Discussion by gaowujian** [umijs/umi#11299] [General]
https://github.com/umijs/umi/issues/6303
todo中提到的分析构建hooks和插件耗时，不知有没有办法进行统计，没找到使用文档
![image](https://github.com/umijs/umi/assets/29998334/e862491a-78d0-4ec6-8d65-f4277d607364)

**Comment by fz6m** [comment]
umi 3 不维护了，这里面不一定都完成了，建议升级到 umi 4
